scripts/kvm-test.py

```diff
@@ -582,6 +582,19 @@ def create_disk(path: Path, size: str):
     run(['qemu-img', 'create', '-f', 'qcow2', str(path), size])
 
 
+def get_grub_appends(ctx, mntdir: str) -> list[str]:
+    with (Path(mntdir) / "boot/grub/grub.cfg").open() as fp:
+        for line in fp:
+            # something like
+            # linux /casper/vmlinuz --- quiet splash
+            expected_sep = " --- "
+            before, actual_sep, after = line.partition(expected_sep)
+            if actual_sep != expected_sep:
```

Comment:
I'd suggest looking for `linux` and `/casper/vmlinuz` tokens as well. If ISOs have multiple boot entries we want the one that matches `linux /casper/vmlinuz` since it is the kernel we're booting.

Maybe `re` will help?

```python
m = re.search(r"linux\s+/casper/vmlinuz\s+---\s+(.*)", line)
print(m.groups(1))
> quiet splash
```

Reply:
implemented, thanks!